src/pages/dmt5/_dhero.js

```diff
@@ -0,0 +1,317 @@
+        top: unset;
+        justify-content: start;
+        margin-top: 40px;
+    }
+`
+
+const StyledLinkButton = styled(LinkButton)`
+    padding: 1.4rem 1.6rem;
+    width: 100%;
+    max-width: 20.2rem;
+    font-size: 14px;
+    max-height: 4rem;
+    height: 100%;
+    margin-right: 2px;
+    border: unset;
+
+    @media ${device.tablet} {
+        padding: 1.5rem 1.6rem;
+        height: 42px;
+        white-space: nowrap;
+        display: inline-block;
+        max-width: 202px;
+        max-height: 40px;
+    }
+`
+const InformationWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 56.2rem;
+
+    @media (max-width: 1376px) {
+        max-width: 44rem;
+    }
+    @media ${device.tabletL} {
+        max-width: 40rem;
+    }
+    @media ${device.tablet} {
+        top: 280px;
+        max-width: 328px;
+        padding: 0 16px;
+    }
+`
+
+const query = graphql`
+    query {
+        dbot: file(relativePath: { eq: "dbot_trade.png" }) {
+            ...fadeIn
+        }
+        dmt5: file(relativePath: { eq: "dmt5_trade.png" }) {
+            ...fadeIn
+        }
+        dtrader: file(relativePath: { eq: "dtrader_trade.png" }) {
+            ...fadeIn
+        }
+        dtrader_mobile: file(relativePath: { eq: "dtrader_trade_mobile.png" }) {
+            ...fadeIn
+        }
+        dbot_mobile: file(relativePath: { eq: "dbot_trade_mobile.png" }) {
+            ...fadeIn
+        }
+        dmt5_mobile: file(relativePath: { eq: "dmt5_trade_mobile.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const DHero = ({
+    title,
+    background_alt,
+    background_svg,
+    background_svg2,
+    content,
+    image_name,
+    is_mobile,
+    join_us_for_free,
+    go_to_live_demo,
+    Logo,
+}) => {
+    const data = useStaticQuery(query)
+    const redirectUrl = () => {
+        const path = image_name === 'dbot' ? '/bot' : '/'
+        return deriv_app_url + path
+    }
+    const DLogo = styled(Logo)`
+        margin-right: 1.6rem;
+    `
+    const BackgroundSVG = styled(background_svg)`
+        position: absolute;
+        top: 170px;
+        right: 0;
+
+        @media ${device.laptopM} {
+            width: 48%;
+            max-width: 492px;
+            height: initial;
+        }
+        @media ${device.laptop} {
+            width: 50%;
+        }
+        @media ${device.tabletL} {
+            width: 45%;
+            max-width: 350px;
+        }
+        @media ${device.tablet} {
+            width: 70%;
+        }
+        @media ${device.tabletS} {
+            width: 80%;
+            max-width: 337px;
+        }
+        @media ${device.mobileL} {
+            max-width: 250px;
+            min-height: 244px;
+        }
+        @media ${device.mobileM} {
+            max-width: 205px;
+            min-height: 0;
+        }
+    `
+    const BackgroundSVG2 = styled(background_svg2)`
+        position: absolute;
+        top: 0;
+        right: 214px;
+
+        @media ${device.laptopM} {
```

Comment:
this is very similar to the one above, you can try using css`` from styled component, and use it both here